### PR TITLE
UI changes from Sept game sessions

### PR DESCRIPTION
**Pull request by dlhoff**

### CODE REVIEW

File: src/conf/gameServer.conf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #This is the configuration file for the game server. 
+# should be 9255, 9272, 9274  for running on wob_wob account
 portNumber 9255
-COSPortNumber 9254
-RRPortNumber 9253
+COSPortNumber 9272
```
Comment (jensvdh) [inline src/conf/gameServer.conf]: Can you please revert those ports?

COS: 9254
RR: 9253

File: src/cvg/net/request/RequestConvergeBetUpdate.java [hunk 2]
```diff
@@ -64,8 +64,9 @@ public void process() throws Exception {
         player.setScores(scores);
         player.setClient(client);
         player.setResponse(response);
+        player.setWinnings(player.getWinnings() - bet);
         
-        Log.println("RCBU: player score status");
+        Log.println("RCBU: player score status");
```
Comment (jensvdh) [inline src/cvg/net/request/RequestConvergeBetUpdate.java]: No need to add whitespace here.

**Review by jensvdh — CHANGES_REQUESTED** [review]
Please fix my proposals and then I can merge this.

File: src/conf/gameServer.conf (hunk 1 above)
Comment (dlhoff) [reply src/conf/gameServer.conf]: Done

File: src/cvg/net/request/RequestConvergeBetUpdate.java (hunk 2 above)
Comment (dlhoff) [reply src/cvg/net/request/RequestConvergeBetUpdate.java]: done

**Pull request merged**